src/js/tabListeners.js

```diff
@@ -274,23 +274,45 @@ function listenForDropdownMenuTabs($scroller, stc) {
           $selectedMenuItemDropdownMenu = $selectedMenuItemLi.parent('.dropdown-menu'),
           targetPaneId = $selectedMenuItemAnc.attr('href');
 
-      if ($selectedMenuItemLi.hasClass('active')) {
-        return;
+      var isActive;
+      if (options.bootstrapVersion == 4) {
+        isActive = $selectedMenuItemLi.hasClass('active');
+      } else {
+        isActive = $selectedMenuItemLi.find('a').hasClass('active');
       }
 
+      if (isActive) {
+         return;
+       }
+
       // once we select a menu item from the dropdown, deactivate
       // the current tab (unless it's our parent tab), deactivate
       // any active dropdown menu item, make our parent tab active
       // (if it's not already), and activate the selected menu item
-      $scroller
-        .find('li.active')
-        .not($ddParentTabLi)
-        .add($selectedMenuItemDropdownMenu.find('li.active'))
-        .removeClass('active');
-
-      $ddParentTabLi
-        .add($selectedMenuItemLi)
-        .addClass('active');
+      if (options.bootstrapVersion == 4) {
+        $scroller
+          .find('li > a.active')
+          .parent('li')
+          .not($ddParentTabLi)
+          .add($selectedMenuItemDropdownMenu.find('li > a.active').parent('li'))
+          .find('a')
+          .removeClass('active');
+
+        $ddParentTabLi
+          .add($selectedMenuItemLi)
+          .find('a')
+          .addClass('active');
```

Comment:
Well-done here in this entire `if` block! 👍 I had to walk myself through it a couple times to make sure I understood what all was going on. Looks really good, though. I trust that it actually works correctly. 😄

Reply:
I just added another example in `run` to check that it works.

BTW it might be worth considering to rename `run` -> `examples` so the devs using the package have an easier time finding the examples.